octoprint_discordremote/discord.py

```diff
@@ -16,7 +16,8 @@
 
 # Constants
 MAX_ERRORS = 25
-CHANNEL_ID_LENGTH = 18
```

Comment:
This is still a useful constant, and should be used to validate that each of the channel IDs are the correct length. 

- [ ] Add constant back in, and validate the channel IDs before use